lucene/core/src/java/org/apache/lucene/index/StoredFieldsConsumer.java

```diff
@@ -42,6 +44,7 @@
   protected void initStoredFieldsWriter() throws IOException {
     if (writer == null) { // TODO can we allocate this in the ctor? we call start document for every doc anyway
       this.writer = codec.storedFieldsFormat().fieldsWriter(directory, info, IOContext.DEFAULT);
+      accountable = writer;
```

Comment:
Hmm what is happening here?  Are we using this new `accountable` member somewhere?  I don't see it in the diffs?  Oh, I see, it is returned in `DefaultIndexingChain.getChildResources()`, ok!  Maybe add comment above its declaration explaining that our parent/owning `DefaultIndexingChain` returns/uses it?